Yoti.Auth.Sandbox/Profile/Request/ExtraData/ThirdParty/SandboxAttributeIssuanceDetailsBuilder.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using Yoti.Auth.Share.ThirdParty;
+
+namespace Yoti.Auth.Sandbox.Profile.Request.ExtraData.ThirdParty
+{
+    public class SandboxAttributeIssuanceDetailsBuilder
+    {
+        private readonly List<AttributeDefinition> _definitions = new List<AttributeDefinition>();
+        private string _issuanceToken;
+        private DateTime? _expiryDate;
+
+        public SandboxAttributeIssuanceDetailsBuilder WithIssuanceToken(string issuanceToken)
+        {
+            Validation.NotNullOrEmpty(issuanceToken, nameof(issuanceToken));
+
+            _issuanceToken = issuanceToken;
+            return this;
+        }
+
+        public SandboxAttributeIssuanceDetailsBuilder WithExpiryDate(DateTime expiryDate)
+        {
+            _expiryDate = expiryDate;
+            return this;
+        }
+
+        public SandboxAttributeIssuanceDetailsBuilder WithDefinition(string attributeName)
+        {
+            Validation.NotNullOrEmpty(attributeName, nameof(attributeName));
+
+            _definitions.Add(new AttributeDefinition(attributeName));
```

Comment:
I think we might want to use a sandbox attribute definition here in case it needs to diverge from the SDK version

Reply:
👍 Fixed in 454545a